WordPress/src/main/java/org/wordpress/android/ui/comments/CommentAdapter.java

```diff
@@ -277,27 +276,37 @@ CommentList getSelectedComments() {
             return comments;
         }
 
-        for (Integer position: mSelectedPositions) {
-            if (isPositionValid(position))
-                comments.add(mComments.get(position));
+        for (Long commentId : mSelectedCommentsId) {
+            if (isCommentIdValid(commentId))
```

Comment:
This is the only place where `isCommentIdValid` is used, and that routine calls `indexOfCommentId` which is again called below. Given that, how about dropping `isCommentIdValid` and using this:
```
for (Long commentId : mSelectedCommentsId) {
    int index = indexOfCommentId(commentId);
    if (index > -1) {
        comments.add(mComments.get(index));
    }
}
```